main.py

```diff
@@ -1,15 +1,22 @@
 import requests
 import json
 import os
+import pathlib
+from config import *
+import datetime
 
-URL = "https://api.divar.ir/v8/web-search/{SEARCH_CONDITIONS}".format(**os.environ)
+URL = f"https://api.divar.ir/v8/web-search/{SEARCH_CONDITIONS}"
 TOKENS = list()
-BOT_TOKEN = '{BOT_TOKEN}'.format(**os.environ)
-BOT_CHATID = '{BOT_CHATID}'.format(**os.environ)
+BOT_TOKEN = f"{BOT_TOKEN}"
+BOT_CHATID = f"{BOT_CHATID}"
```

Comment:
Please keep using the `environment variables` instead of config files.

Reply:
I had a situation where needed to run two instances of this application, 
Using the environment variable is not feasible for this situation so I used a config file instead

Reply:
You can run your code like this to have multiple instances of it:
```bash
SEARCH_CONDITIONS=abc BOT_TOKEN=abc BOT_CHATID=abc python /path-to-file/main.py
SEARCH_CONDITIONS=xyz BOT_TOKEN=xyz BOT_CHATID=xyz python /path-to-file/main.py
```